### PR TITLE
Print waypoint name in istioctl pc workloads

### PR DESCRIPTION
**Pull request by ymesika**
**Please provide a description of this PR:**

As suggested in #43311

### CODE REVIEW

File: releasenotes/notes/43311.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: feature
+area: istioctl
+issue:
+  - 43311
+releaseNotes:
+  - |
+    **Improved** the pc workloads by printing the waypoint name (if any) for the workload.
```
Comment (hanxiaop) [inline releasenotes/notes/43311.yaml:8]:
```suggestion
    **Improved** `istioctl proxy-config workload` by printing the waypoint name (if any) for the workload.
```

**Review by hanxiaop — COMMENTED** [review]
Can you add a test for this?

**Comment by ymesika** [comment]
I can't find an existing test for the pc workloads to update with coverage for my addition.

**Review by ericvn — APPROVED** [review]
Release notes look good.

**Review by howardjohn — APPROVED** [review]

**Review by GregHanson — APPROVED** [review]

**Review by hanxiaop — APPROVED** [review]

**Pull request merged**